packages/beacon-node/src/chain/archiver/index.ts

```diff
@@ -301,21 +303,37 @@ export class Archiver {
       finalizedCanonicalCheckpointsCount,
       finalizedFoundCheckpointsInStateCache,
     });
-    this.logger.info("Attached validators finalized proposal stats", {
-      ...attachedValidators,
-      finalizedAttachedValidatorsCount,
-    });
 
-    this.metrics?.allValidators.expected.set(allValidators.expected);
+    // Only log to info if there is some relevant data to show
+    //  - No need to explicitly track SYNCED state since no validators attached would be there to show
+    //  - debug log if validators attached but no proposals were scheduled
+    //  - info log if proposals were scheduled (canonical) or there were orphans (non canonical)
+    if (finalizedAttachedValidatorsCount !== 0) {
+      if (attachedValidators.total !== 0 || attachedValidators.orphaned !== 0) {
+        this.logger.info("Attached validators finalized proposal stats", {
+          ...attachedValidators,
+          finalizedAttachedValidatorsCount,
+        });
+      } else {
+        this.logger.debug("Attached validators finalized proposal stats", {
+          ...attachedValidators,
+          finalizedAttachedValidatorsCount,
+        });
+      }
```

Comment:
Why do we need to check both `total` and `orphaned`? I would assume orphaned blocks to be counted in total blocks therefore checking if total is 0 should be sufficient.

Potential refactoring/deduplication

```suggestion
      this.logger[attachedValidators.total !== 0 ? LogLevel.debug : LogLevel.info](
        "Attached validators finalized proposal stats",
        {
          ...attachedValidators,
          finalizedAttachedValidatorsCount,
        }
      );
```

Reply:
pls check comment at L310

Reply:
ok maybe I am not understanding this correctly, what does expected (now called total) mean? My naive assumption without looking to deep into the code was it just summing up finalized + missed + orphaned

Reply:
there can be multiple forks in the chain, so the `expected` is as per the canonical chain (and one may have produced blocks in non canonical chain so they are now oprhaned)

for e.g. as per the fork which was finalized, you may not have any proposal, but in the fork in which you proposed, you might have proposed a block which is now orphaned 

so orphaned could be part of expected (for e.g. fork was correct but you send block late which other validators didn't include) or it could be the above scenario.